src/mock_adapter/controllers/product.py

```diff
@@ -3,11 +3,19 @@
 
 import appier
 
+import models
+
 class ProductController(appier.Controller):
 
-    @appier.route("/products/<id>.png", "GET")
+    @appier.route("/products/<int:id>.json", "GET")
+    def show(self, id):
+        product = models.Product.get(id = id, map = True)

```

Comment:
might as well return it directly in this case: return models.Product.get(id = id, map = True)